dist/scss/Components/FormGroup/style.scss

```diff
@@ -0,0 +1,46 @@
+@import './../../vars';
+
+// Normal State
+$label-Dark-Text: #4729b3;
```

Comment:
Letter case `dark`

Reply:
Why did you mark this as resolved?

Reply:
wasnt 'Dark' to be changed to lower case?

Reply:
Yes, but it is still uppercase